src/http/server/cli.rs

```diff
@@ -15,9 +15,8 @@ pub struct HttpServer {
 
     /// Maximum number of HTTP requests to serve over a single connection.
     /// After this number is reached the connection is gracefully closed.
-    /// The default is consistent with nginx's `keepalive_requests` parameter.
-    #[clap(env, long, default_value = "1000")]
-    pub http_server_max_requests_per_conn: u64,
+    #[clap(env, long)]
```

Comment:
What is the default? Should we add that to the help text?

Reply:
There's no default. If it's not set then we don't limit the number of requests.